vello_shaders/shader/coarse.wgsl

```diff
@@ -413,7 +412,12 @@ fn main(
                         if tile.segment_count_or_ix == 0u && tile.backdrop == 0 {
                             clip_zero_depth = clip_depth + 1u;
                         } else {
-                            write_begin_clip();
+                            let blend = scene[dd];
```

Comment:
Handy that this was available. Seeing what the blend mode was at "write begin clip" time was my greatest worry in implementing this (as in I was worried it would be annoying)